### PR TITLE
Add extended key banner to mnemonic flow

### PR DESCRIPTION
**Pull request by Mrtenz**

### CODE REVIEW

**Comment by Mrtenz** [comment]
[ch8441]

File: src/app/components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+const icons: { [key in BannerType]: string } = {
+  success,
+  info,
+  action,
+  warning,
+  error
+};
+
+export const InnerBanner = ({ type, children }: PropsWithChildren<Pick<BannerProps, 'type'>>) => (
+  <Box backgroundColor={theme.variants.banner[type].color} sx={{ borderRadius: 'banner' }}>
```
Comment (FrederikBolding) [inline src/app/components/Banner.tsx:31]: This border radius is not defined.

File: src/app/components/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            sx={{ textTransform: 'uppercase' }}
+          >
+            {label}
+          </Body>
+        </Flex>
+        <Flex variant="horizontal-start">
+          {banner && <InnerBanner type={type}>{banner}</InnerBanner>}
+          {children && (
+            <Box ml="2" sx={{ userSelect: 'none' }}>
+              <Image src={caret} alt="Caret" sx={{ transform: isExtended && 'rotate(180deg)' }} />
```
Comment (FrederikBolding) [inline src/app/components/Banner.tsx:80]: This image is not centered.

File: src/app/components/Banner.tsx (hunk 1 above)
Comment (FrederikBolding) [inline src/app/components/Banner.tsx:31]: We should use the theme from the provider here. Otherwise it will make it difficult to change down the road

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**